package.json

```diff
@@ -12,9 +12,9 @@
     "e2e": "./e2e/start-and-run-suite",
     "e2e:debug": "./e2e/start-and-run-suite debug",
     "e2e:dev": "./e2e/start-and-run-suite dev",
-    "jest": "jest --notify --watch",
+    "test": "jest --notify --watch",
```

Comment:
I would probably do something like `test` and `test:watch`. Personally I usually do `yarn test` just to make sure there are no errors in other parts of code but locally I just run the tests in the IDE and as. With `jest --watch` no tests are run until something changes which usually isn't what I personally want.

Reply:
you can still run all tests by pressing `a`